.buildkite/pipeline.yml

```diff
@@ -0,0 +1,4 @@
+steps:
+  - label: ":pipeline:"
+    commands:
+      - bash .buildkite/pipeline.sh | buildkite-agent pipeline upload
```

Comment:
For our steps that are dynamically generating pipeline steps it can be extremely helpful for debugging to see what the full list of generated steps are. What do you think of adding a `tee` into the piped commands here, @gustavderdrache?

Reply:
> It's always available in the Buildkite UI in the timeline tab
I didn't know it was in there. As we make this more widely used that might be something we want to highlight to assist with debugging efforts. With that available, I agree that the extra output in the console output is no longer necessary.

> you can simply run the script locally via bash .buildkite/pipeline.sh to see the pipeline

I disagree with this given we're customizing the output of that pipeline specifically based on environment details that change depending on where you're running it.